error when compiling opm-common

I'm using RH7 + devtoolset-7

commit > 3a69bcbf4adbd844935872f1e800bb19b245ae0b ?? Merge pull request #1503 from joakim-hove/wellsegments-alt-cons

compiles without error

next commit 

> 09595b98adb97b79c9147af69a75b492b0c81b56 Merge pull request #1495 from joakim-hove/opmpack-copy-restart

compiles with the following error

/project/multiscale/OPM/tskille/flow_daily_rh7/opm-common/examples/opmpack.cpp: In function 'void copy_file(const std::experimental::filesystem::v1::path&, std::experimental::filesystem::v1::path, const std::experimental::filesystem::v1::path&)':
/project/multiscale/OPM/tskille/flow_daily_rh7/opm-common/examples/opmpack.cpp:79:21: error: 'relative' is not a member of 'fs'
         fname = fs::relative(fname, source_dir);
                     ^~~~~~~~
make[2]: *** [CMakeFiles/opmpack.dir/build.make:63: CMakeFiles/opmpack.dir/examples/opmpack.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:2206: CMakeFiles/opmpack.dir/all] Error 2

Other who have the same issue ?

--- a/examples/opmpack.cpp
+++ b/examples/opmpack.cpp
@@ -75,8 +75,14 @@ but restart and import files referred to in the deck are also copied. The -o and
 
 
 void copy_file(const fs::path& source_dir, fs::path fname, const fs::path& target_dir) {
-    if (fname.is_absolute())
-        fname = fs::relative(fname, source_dir);
+    if (fname.is_absolute()) {
+        // change when moving to gcc8+
+        // fname = fs::relative(fname, source_dir);
+        auto cpath = fs::current_path();
+        fs::current_path(source_dir);
+        fname = fname.relative_path();
+        fs::current_path(cpath);
+    }
 
     auto source_file = source_dir / fname;
     auto target_file = target_dir / fname;

